doc/lua_api.txt

```diff
@@ -2580,6 +2580,8 @@ These functions return the leftover itemstack.
     and `reconnect` == true displays a reconnect button.
 * `minetest.get_server_status()`: returns server status string
 * `minetest.get_server_uptime()`: returns the server uptime in seconds
+* `minetest.remove_player(name)`: remove player from database (if he is not connected).
+    * Returns a code (0: removed, 1: no_such_player, 2: error_connected)
```

Comment:
Lua can check whether the player is online beforehand, no need for 3 different return codes